modules/example-browser/src/fallback.ts

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ * this file except in compliance with the License. A copy of the License is
+ * located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* This is a simple example of using a raw AES keyring
+ * to encrypt and decrypt using the AWS Encryption SDK for Javascript
+ * in a browser.
+ */
```

Comment:
Specifically, this example shows how to do that while providing a fallback to use in the event of some missing WebCrypto features. That should be mentioned in this intro.

Reply:
fixed